content/pt/docs/reference/glossary/cluster.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Cluster
+id: cluster
+date: 2020-08-03
+full_link: 
+short_description: >
+   Um conjunto de servidores de processamento, também chamados de "nodes", que executam aplicações containerizadas. Todo cluster possui ao menos um "worker node" (servidor de processamento).
+   
+aka: 
+tags:
+- fundamental
+- operation
+---
+Um conjunto de servidores de processamento, chamados {{< glossary_tooltip text="nodes" term_id="node" >}}, que executam aplicações containerizadas. Todo cluster possui ao menos um "worker node" (servidor de processamento).
```

Comment:
nit: I'd mention nós on this line.

Reply:
Okay, I've kept this as 'node' originally because the glossary from 'node' also calls it as 'node': 

```
short_description: >
  Um Node é uma máquina de trabalho no Kubernetes.
```

IMO, this can be kept as node (as a technical term) or maybe translate all the occurrences to "nó", WDYT?

Reply:
I've put 'nó' as the tooltip text

Reply:
Ricardo, I agree with Tim @sftim, we need more Portuguese than English terms. Replace all Node term to "Nó", should be better always keep more Portuguese than English, when someone prefers to read in English, he's can change the language.